api-channel-issue-tracker/src/main/java/com/synopsys/integration/alert/api/channel/issue/model/IssueBomComponentDetails.java

```diff
@@ -28,6 +29,7 @@ public static IssueBomComponentDetails fromSearchResults(LinkableItem component,
             List.of(),
             UNKNOWN_LICENSE,
             UNKNOWN_USAGE,
+            ComponentUpgradeGuidance.none(),
```

Comment:
If this is not required in this class, then let's not include it in the abstract class.

Reply:
It's still required for the other static method, fromBomComponentDetails